### DISCUSSION TITLE
How to reinstall a flux repo on a new server?

### DISCUSSION THREAD

**Discussion by baflo** [fluxcd/flux2#2976] [Q&A] [answered]
Hey,

I am working with fluxcd for a couple of week. I am planning how to move a fluxcd repo from one server to another. Now I am wondering how exactly the repo is bootstrapped and how I would best apply the manifests on a new server.

After bootstrapping as described in [Get started](https://fluxcd.io/docs/get-started/), I end up with the following files:

```
└── clusters
    └── my-cluster
        └── flux-system
            ├── gotk-components.yaml
            └── gotk-sync.yaml
```

In `gotk-sync.yaml` I find a `GitRepository` pointing to the very same repository as the files themselves are placed including a `secretRef`. 

When I set up a new server, how would I best prepare this secret?

**Comment by stefanprodan** [comment, marked as answer]
You can rerun `flux bootstrap` on the new server and the secret will be generated in the cluster and the deploy key will be updated in the repo.
